src/local-mode.ts

```diff
@@ -142,6 +148,39 @@ export class LocalModeManager {
 		);
 	}
 
+	private async collectContainerResources(
+		nameOrId: string,
+	): Promise<EngineSnapshot> {
+		const inspectInfo = await this.docker.getContainer(nameOrId).inspect();
+		return new EngineSnapshot(
+			[inspectInfo.Id],
+			[inspectInfo.Image],
+			inspectInfo.Mounts.filter(m => m.Name !== undefined).map(m => m.Name!),
+			_.values(inspectInfo.NetworkSettings.Networks).map(ns => ns.NetworkID),
```

Comment:
```suggestion
			_.mapValues(inspectInfo.NetworkSettings.Networks, 'NetworkID'),
```

Reply:
It's not type safe! :)
Do you insist?

Reply:
Hm I thought it was type safe? Otherwise I'd prefer `mapValues` with a lambda

Reply:
I'd go for
```suggestion
			_.mapValues(inspectInfo.NetworkSettings.Networks, ns => ns.NetworkID),
```
to get the best of both

Reply:
In fact, `mapValues` transforms object to an object - it does not return an array
https://lodash.com/docs/4.17.14#mapValues

Reply:
Right, I was just looking at the type safe bit, you can use straight `_.map` in that case to have an array returned

Reply:
Changed.